prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -17,10 +17,15 @@ if [[ $2 != "LOCAL" && ! -f $2 ]]; then
   exit 1
 fi
 
+if [[ $1 == "CTF" ]]; then
+  export EXTINPUT=0
+fi
+
```

Comment:
I am a bit confused by this. `EXTINPUT` is only set to 1 in case `if [[ $1 == "DD" ]]; then` below.
So what is this needed for? It would only make sense in case the user would set EXTINPUT=1 on the command line when invoking this script. That would actually indeed break it, but then there are 10 other env variables that could break it if the user sets them. So I am not sure this single check here has much benefit.
What we should perhaps do is define a list of env variables that are set by this script ( e.g. `EXTINPUT`, `CTFINPUT`, `RAWTFINPUT`, `INPUT_FILE_LIST`) and that should not be set by the user, and then check if any of them are set at startup, and if so abort with an error.

Reply:
We just had the problem that even when we ran with `CTF` mode then we got the error message `EXTINPUT and CTFINPUT are incompatible`, because when running on the EPNs EXTINPUT is set to 1 by default (if it is not set before) in setenv.sh

Reply:
OK, now I am confused. Did you perhaps set EPNMODE=1, which sets some defaults for the synchronous processing, like to use 4 GPUs per NUMA domain and it also enables the EPNPIPELINES?
That would explain why EXTINPUT was set for you, and that would also explain some of the !=1 pipeline settings you had. If that was the case, sorry for the confusion :(
Perhaps we should rename that in EPNSYNCMODE=1...

Reply:
Actually yes, we naively set `EPNMODE=1`, because we were running on the EPN... We will change this then

Reply:
OK, I have renamed it in #7516 and updated the READMEs to make it more clear.
In order to use it, you will also need the new O2DataProcessing with new setenv, I have bumped it in alidist.